src/main/resources/l10n/JabRef_it.properties

```diff
@@ -2275,13 +2275,13 @@ last_four_nonpunctuation_characters_should_be_numerals=gli_ultimi_quattro_caratt
 shared=condiviso
 should_contain_an_integer_or_a_literal=deve_contenere_almeno_un_intero_o_una_lettera
 should_have_the_first_letter_capitalized=la_prima_lettera_deve_essere_maiuscola
-Tools=
-What\'s_new_in_this_version?=
-Want_to_help?=
-Make_a_donation=
-get_involved=
-Used_libraries=
-MIT=
+Tools=Strumenti
+What\'s_new_in_this_version?=Cosa_c\'è_di_nuovo_in_questa_versione?
+Want_to_help?=Ti_serve_aiuto?
+Make_a_donation=Fai_una_donazione
+get_involved=collabora_con_noi
+Used_libraries=Librerie_usate
+MIT=MIT
```

Comment:
My first guess it is used somewhere in the About dialog. The license. But makes no sense to have it in the localization